ansible/image-upload.yml

```diff
@@ -0,0 +1,48 @@
+---
+
+- name: Push Collector Image to Hosts
+  vars_files:
+    - group_vars/all.yml
+  hosts: "platform_*:&job_id_{{ job_id }}"
+  strategy: free
+
+  tasks:
+    - name: Get Collector Image
+      shell: make -C "{{ collector_root }}" tag
+      register: collector_tag_result
+      delegate_to: localhost
+
+    - set_fact:
+        collector_tag: "{{ collector_tag_result.stdout | replace('\n', '') }}"
+
+    - set_fact:
+        collector_tag: "{{ lookup('env', 'COLLECTOR_TAG') | default(collector_tag) }}"
```

Comment:
Looks kinda weird that we are setting `collector_tag` twice, maybe we could save the filtered one to a different fact?